src/main/java/HttpServerThread.java

```diff
@@ -0,0 +1,48 @@
+package com.td.HttpServer;
+
+public class HttpServerThread implements Runnable {
```

Comment:
If it helps, think about what would you have to do if I asked you for an easy way to configure this server to be multithreaded or not?

Ideally, you would change one line of code to enable/disable that feature (hint: instantiate _this_ guy for single threading or _that_ guy for multithreading).